src/languageProvider/locales/en_US.json

```diff
@@ -73,6 +73,7 @@
   "oracle.consensus": "Consensus Threshold {value}. This value indicates the amount of BOT needed to set the result.",
   "oracle.enterAmountDisabledText": "Please entered a valid amount.",
   "oracle.eventUnconfirmed": "This created Event is unconfirmed. You cannot interact with it until it is confirmed by the blockchain.",
+  "oracle.optionIsPrevResult": "This option was set as result in previous rounds",
```

Comment:
"This option was set as the result in the previous round"
- added "the"
- use singular round, not rounds. as we only disable the result of the previous round.